src/data/resolvers/colonyActions.ts

```diff
@@ -9,6 +9,7 @@ import {
 import { BigNumberish } from 'ethers/utils';
 import { AddressZero } from 'ethers/constants';
 import { Resolvers } from '@apollo/client';
+import uniqWith from 'lodash/uniqWith';
```

Comment:
This belongs in ~utils/lodash. Makes imports neater in the event you have two or more from lodash.